### DISCUSSION TITLE
Dear ImGui git history visualization

### DISCUSSION THREAD

**Discussion by Demonese** [ocornut/imgui#3859] [Old/Archived discussions]
I use gource made a git history visualization video ([Youtube](https://www.youtube.com/watch?v=JE0BwK-5mwY)).

HTML (Seems not working...):
```html
<iframe width="560" height="315" src="https://www.youtube.com/embed/JE0BwK-5mwY" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Markdown clickable image:
```
[![Dear ImGui git history visualization](http://img.youtube.com/vi/JE0BwK-5mwY/0.jpg)](http://www.youtube.com/watch?v=JE0BwK-5mwY "Dear ImGui git history visualization")
```
[![Dear ImGui git history visualization](http://img.youtube.com/vi/JE0BwK-5mwY/0.jpg)](http://www.youtube.com/watch?v=JE0BwK-5mwY "Dear ImGui git history visualization")

**Comment by ocornut** [comment]
Nice! Thanks!

It's a little odd that github magically "merges" local git user names based on matching e-mails or multiple e-mails, but most tools don't as they don't have the info. As a result for a long time my git (not github) name was different depending on my workstation and I'm seen with a split persona :)
